contents/blog/the-posthog-array-1-36-0.mdx

```diff
@@ -0,0 +1,157 @@
+    src="https://www.youtube.com/embed/clVpovL0vxI"
+    title="YouTube video player"
+    frameborder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+    allowfullscreen
+></iframe>
+
+[Restack](https://posthog.com/marketplace/restack) is a platform for deployiong open-source software stacks on Kubernetes and is run by a team of open-source enthusiasts. It's a natural match for [the PostHog Marketplace](https://posthog.com/marketplace/), where we've steadily grown the number of businesses offering services and support over the last few months.
+
+Restack enables you to deploy and benefit from a self-hosted PostHog instance, but without the maintenance busy work. Best of all, Restack Console can ensure that deployments are [HIPAA](/docs/privacy/hipaa-compliance), FISMA and SOC2 compliant — handy if you deal with a lot of [PII](/blog/what-is-personal-data-pii). Find out more in [the Marketplace, or head to [Restack's store listing for PostHog](https://www.restack.io/store/posthog) to find more great videos like the one above!
+
+### Other improvements & fixes
+
+Version 1.36 also adds hundreds of other improvements and fixes, including...
+
+-   **New**: Median conversion time is now presented in funnel insights, offering results that are more resistant to being skewed by outliers. The average continues to be available in detailed results. [#9810](https://github.com/PostHog/posthog/pull/9810)
+-   **Improved**: The in-app plugin source code editor has been divided into tabs, to make editing `plugin.json` and `index.ts` slightly more intuitive. Additionally, code is now automatically formatted with `prettier` on save. [#9795](https://github.com/PostHog/posthog/pull/9795)
+-   **Improved**: Large numbers are now presented with the comma as the thousands separator in Insights, greatly improving readability. [#9725](https://github.com/PostHog/posthog/pull/9725) [#9733](https://github.com/PostHog/posthog/pull/9733)
+-   **Improved**: Properties table column sizing has been made smarter, making them scale better for various contents of properties. [#9502](https://github.com/PostHog/posthog/pull/9502)
+-   **Fixed**: User Paths with start/end points containing trailing slashes work now. It's also been made impossible to unselect all event categories for a more intuitive experience. [#9482](https://github.com/PostHog/posthog/pull/9482)
+-   **Fixed**: Trends insight displayed as a table can be sorted indepentently on dashboards. [#9625](https://github.com/PostHog/posthog/pull/9625)
+-   **Fixed**: Custom series names are included in CSV exports. [#9677](https://github.com/PostHog/posthog/pull/9677)
+-   **Fixed**: Automatic provisioning of PostHog users logging in with SSO has been made more reliable. For instance, users who first logged-in _before_ automatic provisioning was turned on for the organization, will automatically added to it. [#9515](https://github.com/PostHog/posthog/pull/9515)
```

Comment:
CC @mariusandra Is this completely correct?

Reply:
Almost:

```suggestion
-   **Fixed**: Automatic provisioning of PostHog users logging in with SSO has been made more reliable. For instance, users who first logged-in _before_ automatic provisioning was turned on for the organization, will be automatically added to it when they log in again. [#9515](https://github.com/PostHog/posthog/pull/9515)
```